### ISSUE TITLE
Allow Different Shells to be Tested Against

### ISSUE THREAD

**Issue by Justintime50** [ludeeus/action-shellcheck#34]
## Checklist

- [x] This Feature Request only contains 1 request (if you have multiple open multiple feature requests).

## The idea

I want to be able to tell shellcheck to check my script against different shells, (eg: `shellcheck dir/*.sh --shell=sh`). I want to be able to specify which shell to check against. Shellcheck allows the following inputs when using this flag:

* `bash`
* `sh`
* `dash`
*  `ksh`

## Implementation

Add an input to this action such as `shell` which takes one of the four supported shells to check against mentioned above.

## Alternatives

NA

## Additional context

NA

**Comment by ludeeus** [comment]
You can do that already

```yaml
    ...
    - name: Run ShellCheck
      uses: ludeeus/action-shellcheck@master
      env:
        SHELLCHECK_OPTS: -s bash
```

```yaml
    ...
    - name: Run ShellCheck
      uses: ludeeus/action-shellcheck@master
      env:
        SHELLCHECK_OPTS: -s sh
```

```yaml
    ...
    - name: Run ShellCheck
      uses: ludeeus/action-shellcheck@master
      env:
        SHELLCHECK_OPTS: -s dash
```

```yaml
    ...
    - name: Run ShellCheck
      uses: ludeeus/action-shellcheck@master
      env:
        SHELLCHECK_OPTS: -s ksh
```
